app/models/service/MemberService.scala

```diff
@@ -82,7 +82,7 @@ class MemberService {
    * @param objectIds List of identifiers
    */
   def find(objectIds: List[Long]): List[Member] = DB.withSession { implicit session =>
-    members.filter(_.objectId inSet objectIds).list
+    members.filter(_.id inSet objectIds).list
   }
```

Comment:
You cannot just replace _.objectId to _.id. This method is used in MemberService, line 154. It's better to rename this service to findByObjectId and add a new one.